src/library/handlers/logHandler.ts

```diff
@@ -144,6 +164,20 @@ export class LogHandler {
         try {
             const envelope = this._exceptionToEnvelope(telemetry, this.config.instrumentationKey);
             this._batchProcessor.send(envelope);
+
+            if (this._metricHandler?.isStandardMetricsEnabled) {
+                let exceptionDimensions: IMetricExceptionDimensions = {
+                    cloudRoleInstance: envelope.tags[KnownContextTagKeys.AiCloudRoleInstance],
+                    cloudRoleName: envelope.tags[KnownContextTagKeys.AiCloudRole]
```

Comment:
Another field `client/type` might be needed. Curious did you look into what this dimension represents?

Reply:
I was not aware of the existence of that dimension, currently only enabling what we had before